### DISCUSSION TITLE
Migrating from v1 to v2

### DISCUSSION THREAD

**Discussion by kanisornboo** [parcel-bundler/parcel#5491] [General]
Hi, 

I am trying to migrate current parcel version 1 to use the latest and greatest feature in version 2 but none of the old cofig seems to be working. can anyone help? When I ran the script below via cmd I get Parcel is not a constructor. What am I missing here?

```js
// const Parcel= require('parcel-bundler') - old version
const Parcel= require('@parcel/core') 

const options = {
  production: true,
  minify: true,
  outputfile: 'index.js'
}
const jsBundler = new Parcel('./src//index.js', options)
```

**Comment by mischnic** [comment]
Have you seen https://v2.parceljs.org/features/parcel-api/? With CommonJS, you probably need to do `const {default: Parcel} = require('@parcel/core') `

**Reply by kanisornboo** [reply]
I haven't yet but will take a look at it now. Thank you! 😎

**Reply by kanisornboo** [reply]
I also using .on(''buildEnd) is not avaible for uses? I keep getting jsBundler.on('buildNed', ()=> {}) is now a function.

e.g

``` js
const jsBundler = new Parcel('./src//index.js', options)
 jsBundler.on('buildEnd', ()=> {
 cssBundler.bundle()
}
```

**Reply by jsjoeio** [reply]
@kanisornboo did you ever figure this out? We also use `buildEnd`
